src/Home/Home.tsx

```diff
@@ -1,16 +1,28 @@
 import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import DisplayBackgroundImage from '../images/DisplayBackgroundImage';
 
-function Home() {
+interface getUserProp{
+    username:string;
+    setUsername:(uname:string)=>void;
+}
+
+function Home(props:getUserProp) {
 
-    const [username, setUsername] = useState("");
+    //const [username, setUsername] = useState("");
```

Comment:
there are a couple of lines commented out in this file that I think are old code that we're not using.
Let's remove them as it could cause confusion when looking back or editing this code.